corehq/apps/reports/views.py

```diff
@@ -940,9 +942,11 @@ def scheduled_report_form(self):
                 web_user_emails = [email] + web_user_emails
 
         from corehq.apps.reports.forms import ScheduledReportForm
+        enable_start_date = toggles.SET_SCHEDULED_REPORT_START_DATE.enabled(kwargs['initial']['domain'])
```

Comment:
What's `kwargs['initial']['domain']`? I don't see a domain input in this form. `ScheduledReportsView` probably has a `domain` attribute you can use here.